src/platform/Linux/BLEManagerImpl.cpp

```diff
@@ -735,7 +735,7 @@ void BLEManagerImpl::NewConnection(BleLayer * bleLayer, void * appState, const S
 CHIP_ERROR BLEManagerImpl::CancelConnection()
 {
     if (mBLEScanConfig.mBleScanState == BleScanState::kConnecting)
```

Comment:
should check that mEndpoint is a valid pointer and return a CHIP_ERROR otherwise.

Reply:
Done